exercises/bank-account/uBankAccountTests.pas

```diff
@@ -37,16 +37,19 @@ implementation
 
 procedure BankAccountTests.Returns_empty_balance_after_opening;
 var account: IBankAccount;
+    Expected: double;
 begin
+  Expected := 0.00;
```

Comment:
After the `0.0` is there a significance to the additional `0`?  Should this be `0.0`?

It seems "unnatural" to see it here and in the other places.